nemo/collections/llm/recipes/mistral.py

```diff
@@ -32,7 +32,7 @@ def trainer(devices=8) -> nl.Trainer:
 
 @factory(name=NAME + "_hf")
 def hf_resume() -> nl.AutoResume:
-    return nl.AutoResume(import_path="hf://mistralai/Mistral-7B-v0.3")
+    return nl.AutoResume(selective_restore_config=nl.SelectiveRestoreConfig(path="hf://mistralai/Mistral-7B-v0.3"))
```

Comment:
why isn't this `Config(nl.SeclectiveRestoreConfig ...` ?

Reply:
Looks like the recipes in this file are not updated. MIght require a separate PR to update all recipes here.